### ISSUE TITLE
Please make this happen.

### ISSUE THREAD

**Issue by rohangiriraj** [geohot/fromthetransistor#10]
Once you're done with reverse engineering the corona virus, please make this course happen.

**Comment by hotwaterheater** [comment]
Yes! I'd be super pumped to see this course become something I can go through. Also you don't have to come up with all the content yourself, you could just refer to sections of other peoples courses and build it up like that (at least in the beginning).

**Comment by x-N0** [comment]
Struggle to learn and experimentation is the most ancient and the best way to put new info in our brains, so just start working on the README by yourself. You'll get a richer experience going by yourself because your brain will struggle to cheat the way you learn, achieving a faster learning plan in the process (mostly finding patterns inside the contents), always ask for the Why's.
That's my advice, guys.
